### DISCUSSION TITLE
Show data from specific month in table

### DISCUSSION THREAD

**Discussion by Pieter0313** [Budibase/budibase#5692] [Help] [answered]
Hi! I'm new to budibase and I would like to only show the data that have the current month in the 'date' column. I think the 'filtering' section is where I can do that, but I don't really know how.. Would be great if someone can help me out :)
<img width="990" alt="image" src="https://user-images.githubusercontent.com/52565351/166487100-74b1884e-3087-4747-bbe7-ef7f72325c0a.png">

**Comment by melohagan** [comment, marked as answer]
Hey @Pieter0313 

You're in the right place, but I think you'll need to use a **static** formula column for this.
Add a column called *Month* that is a static formula that will format your date field as follows:
`{{ date Betaaldatum "MMM" }}`

More here: https://docs.budibase.com/docs/formula

Then you can use an *equals* filter on the new field against the following handlebars expression: 
`{{ date now "MMM" }}`

**Reply by Pieter0313** [reply]
Hi @melohagan, thank you very much for the reply. I did not know that was possible, that's actually really useful! I created the 'Month' column but for some reason it won't show up in the columns list in the Filtering section (see screenshot). Do you have any idea why that is?
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/52565351/166564019-44c9105d-3afb-4642-967c-8aa41a6d2977.png">

**Reply by Pieter0313** [reply]
I now see that I selected 'Dynamic' instead of 'Static' as formula type, my bad! It works now, so thank you very much! :)

**Comment by tomy137** [comment]
Hello ! 

How can I manage to do exactly the same thing but with external database ? In this case, I can't write static formula and classic formula don't appear in filters list.

Thanks in advance !

**Reply by ConorWebb96** [reply]
Hey @Pieter0313,

Which external db are you using?
If you are using a query based e.g. postgres, MySQL, MsSQL you should use the custom queries functionality for this instead.

Additionally, there is a open issue with formulas and external database at the moment you can find this issue [here](https://github.com/Budibase/budibase/issues/11972). Once this has been resolved you should be able to do the exact same approach as seen in the Answer provided by Mel.

I hope this helps.

**Reply by tomy137** [reply]
I am using MySQL. So have I to create a custom query to add manually month column ?
